### ISSUE TITLE
Cacti scores low on performance audit on lighthouse audit

### ISSUE THREAD

**Issue by TheWitness** [Cacti/cacti#3402]
**Describe the bug**
Cacti is scoring low from a performance perspective on the lighthouse audit available in Chrome.  This bug will cover a set of changes that improve that score, but may not resolved all issues.

**To Reproduce**
1. Open the Cacti website using chrome
2. Press F-12 on Windows to bring up developers tools
3. Goto the Audit tab and run the audit
4. See score

**Expected behavior**
Cacti should behave as best as possible, knowing that some performance hit's can not be addressed except by the customer due to the OSS nature of the tool.

**Screenshots**
![image](https://user-images.githubusercontent.com/1439914/78451120-88eec780-7651-11ea-858e-5970edb12845.png)

**Comment by TheWitness** [comment]
New score after tuning.

![image](https://user-images.githubusercontent.com/1439914/78451547-3367ea00-7654-11ea-931d-7bbfe7b210fb.png)

**Comment by TheWitness** [comment]
I think the performance number is deceptive, since the items it's complaining about are already cached.  So, it's non-blocking.  But this is better than it was when we started and I've see some pages with performance numbers closer to 44.

Reading through the Accessibility score, I did find some real target's for improvement, but those will have to take place in the 1.3+ timeframe, since they have a large scope.

![image](https://user-images.githubusercontent.com/1439914/78455156-a2503d80-766a-11ea-8af7-9f023da783cc.png)
